packages/react/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@logto/react",
+  "version": "0.1.0",
+  "source": "src/index.ts",
+  "main": "lib/index.js",
+  "exports": "./lib/index.js",
+  "typings": "lib/index.d.ts",
+  "license": "MPL-2.0",
+  "scripts": {
+    "dev:tsc": "tsc -p tsconfig.build.json -w --preserveWatchOutput",
+    "preinstall": "npx only-allow pnpm",
+    "precommit": "lint-staged",
+    "build": "tsc -p tsconfig.build.json",
+    "lint": "eslint --ext .ts --ext .tsx src",
+    "package": "parcel build",
+    "test": "jest",
+    "test:coverage": "jest --silent --coverage",
+    "test:report": "codecov -F react",
+    "prepublish": "pnpm build && pnpm package",
+    "report": "WITH_REPORT=true pnpm package"
+  },
+  "dependencies": {
+    "@logto/browser": "^0.1.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-router-dom": "^6.2.1"
+  },
+  "devDependencies": {
+    "@jest/types": "^27.5.1",
+    "@parcel/core": "^2.3.2",
+    "@silverhand/eslint-config": "^0.9.1",
+    "@silverhand/eslint-config-react": "^0.9.1",
+    "@silverhand/ts-config": "^0.9.1",
+    "@silverhand/ts-config-react": "^0.9.1",
+    "@testing-library/react": "^12.1.3",
+    "@testing-library/react-hooks": "^7.0.2",
```

Comment:
testing library looks not needed for now?

Reply:
Removed for now. PTAL